client/src/pages/CreateEvent.js

```diff
@@ -0,0 +1,227 @@
+                    </DialogTitle>
+                    <DialogContent>
+                        <DialogContentText className={classes.instruction}>
+                            Please choose how long your new event type will be.
+                        </DialogContentText>
+                        <Grid
+                            container={true}
+                            direction="row"
+                            alignItems="baseline"
+                            justify="center"
+                            spacing="2"
+                            className={classes.groupedInput}
+                        >
+                            <Grid item>
+                                <Input
+                                    autoFocus
+                                    label="Duration"
+                                    type="number"
+                                    onChange={handleDurationChange}
+                                    disableUnderline={true}
+                                />
+                            </Grid>
+                            <Grid item>
+                                <Select
+                                    value={unit}
+                                    onChange={handleUnitChange}
+                                    label="Units"
+                                    disableUnderline={true}
+                                >
+                                    <MenuItem value={"min"}>minutes</MenuItem>
+                                    <MenuItem value={"hour"}>hours</MenuItem>
+                                </Select>
+                            </Grid>
+                        </Grid>
+                    </DialogContent>
+                    <DialogActions>
+                        <Grid
+                            container={true}
+                            alignItems="baseline"
+                            justify="flex-end"
+                        >
+                            <Button
+                                onClick={handleClose}
+                                color="primary"
+                                className={classes.cancel}
+                            >
+                                Cancel
+                            </Button>
+                            <Button
+                                onClick={createNewEventType}
+                                color="primary"
+                                className={classes.button}
+                            >
+                                Create New Event Type
+                            </Button>
+                        </Grid>
+                    </DialogActions>
+                </Box>
+            </Dialog>
+        </>
+    );
+}
+
+function MeetingCard({ type }) {
+    function addAppointment() {
+        axios
+            .post("/api/appointment", {
+                // dummy data
+                meeting_id: new mongoose.Types.ObjectId(),
```

Comment:
This line isn't needed and you can avoid importing mongoose on the frontend. If you want to pass the id for the meeting type, that can be passed as a string from the front end and the backend can use that to link to the meeting type `mongoose.Types.ObjectId(string)`

Reply:
I had to generate them to get the call to work because they have to be unique ids or the post will fail, I can leave it as non-working until we decide on how to generate meeting ids on the backend